### ISSUE TITLE
Lab Report 5 Feedback

### ISSUE THREAD

**Issue by gdietz8** [lvuluong/cse15l-lab-reports#5]
Here is your feedback for Lab Report 5:

- [x] Explains how tests with different results were found (`diff`, manual search, etc)

Test 1:

- [x] Describes which, if any, implementation is correct by showing both actual outputs and indicating what the expected output is
- [x] Describes bug in code for one of the incorrect implementation and shows the code that should be fixed.

Test 2:

- [x] Describes which, if any, implementation is correct by showing both actual outputs and indicating what the expected output is
- [x] Describes bug in code for one of the incorrect implementation and shows the code that should be fixed.
  - make sure to show the code that should be fixed for this bug! (the line 57 you mentioned isn't in the screenshot for the code for test 1)

Additional comments from your grader:
Great job!

Feel free to comment below any questions about the rubric/feedback

--------

If you don't have all the checkmarks above, fix the issues described, and then submit a [regrade request on Gradescope](https://help.gradescope.com/article/8hchz9h8wh-student-regrade-request). Use this template for your regrade request:

- Link to this issue
- Link to your published lab report page

**Comment by lvuluong** [comment]
Hi, so I just want to make sure that the reason I didn't get the last checkmark is that I need to show the screenshot for the bug of test 2? So is my explanation of the bug correct then or do I also need to reword that?

**Comment by gdietz8** [comment]
Yes, your explanation is perfect, so you don't need to reword anything! Just include the screenshot of the code you're referring to!

**Comment by lvuluong** [comment]
I see thank you, also I have finished the revision so you can check my revision now since the grade are not in on Gradescope so I can't put in a regrade request. And please let me know if I need further revision

**Comment by gdietz8** [comment]
Sounds good, I can do that!
